### DISCUSSION TITLE
mysql巡检报告

### DISCUSSION THREAD

**Discussion by czxin788** [hhyo/Archery#1637] [General]
1、脚本使用方法
mysql -h192.168.1.x  -uroot -p123 -P3306 --html -t -f --silent < mysql巡检脚本.sql
2、执行完后会在当前目录下生成一个html文件，这个html就是巡检报告
3、详情见附件
[巡检脚本和示例.zip](https://github.com/hhyo/Archery/files/8947008/default.zip)

**Comment by linux-y** [comment]
这个巡检脚本哪来的？
